### PR TITLE
Feat.Labels

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Labels

### CODE REVIEW

File: src/frontend/components/NoteCard/NoteCard.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ const NoteCard = ({ note }) => {
     editNoteHandler(e, { ...note, bgColor: color });
   }
 
-  console.log(existInArchive);
+  // console.log(existInArchive);
```
Comment (Chakravarthi0) [inline src/frontend/components/NoteCard/NoteCard.js:43]: You can remove this comment

File: src/frontend/components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -1,16 +1,18 @@
 .sidebar {
-  width: 12rem;
-  background-color: var(--bg2-color);
+  min-width: 14rem;
+  /* background-color: var(--bg2-color); */
```
Comment (Chakravarthi0) [inline src/frontend/components/Sidebar/Sidebar.css:3]: This one as well

File: src/frontend/context/notesContext.jsx [hunk 3]
```diff
@@ -30,6 +27,7 @@ const NotesProvider = ({ children }) => {
     NotesReducerFun,
     InitialValues
   );
+
   // console.log(notesState.trashList)
```
Comment (Chakravarthi0) [inline src/frontend/context/notesContext.jsx:31]: You can remove this comment

File: src/frontend/context/notesContext.jsx [hunk 4]
```diff
@@ -104,7 +102,7 @@ const NotesProvider = ({ children }) => {
       (async function () {
         try {
           const { status, data } = await posttrashnoteService(token, note);
-          console.log(data,"from tash");
+          console.log(data, "from tash");
```
Comment (Chakravarthi0) [inline src/frontend/context/notesContext.jsx:105]: this one too

**Review by Chakravarthi0 — APPROVED** [review]
Code looks clean.

you have few comments and console.log statements which can be removed

rest looks good

**Comment by RahulKuvlekar** [comment]
Good to Merge 👍🏻

**Review by preethasri — APPROVED** [review]
code looks clean . good to merge .UI looks great one suggestion is if i signup it is again redirecting to login instead it can be redirected to home page.

**Comment by Shrinkhlarajpoot** [comment]
> code looks clean . good to merge .UI looks great one suggestion is if i signup it is again redirecting to login instead it can be redirected to home page.

I have made its functionality like this

**Pull request merged**